tox.ini

```diff
@@ -4,38 +4,48 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py27, py34, py35, py36, py37
+envlist = py27, py35, py36, py37, {py35, py36, py37}-async
 
 [testenv]
 commands = coverage erase
-           coverage run -m unittest discover -v []
-           coverage report
+           coverage run -m unittest discover -p "test_*.py" -v []
+           coverage report --ignore-errors
```

Comment:
Why `--ignore-errors`?

Reply:
Without `--ignore-errors` `coverage` for Python 2.7 fails because of `invalid syntax`.